app/uk/gov/hmrc/serviceconfigs/controller/DeploymentConfigController.scala

```diff
@@ -47,6 +49,26 @@ class DeploymentConfigController @Inject()(
         })
     }
 
+  def deploymentConfigGrouped(serviceName: Option[ServiceName], teamName: Option[TeamName], repoType: Option[String], sort: Option[String]): Action[AnyContent] =
```

Comment:
Deployment configs only apply to Services - so the `repoType` param will be confusing.

`sort` is unnecessary since the client can sort trivially.